lib/kitchen/directions/bake_chapter_summary.rb

```diff
@@ -10,7 +10,8 @@ def self.v1(chapter:, metadata_source:)
         # TODO: include specific page types somehow without writing it out
         chapter.pages('$:not(.introduction)').each do |page|
           summary = page.summary
-          summary.first("[data-type='title']").trash # get rid of old title
+          summary.first("[data-type='title']").trash if
+            summary.first("[data-type='title']").respond_to? :trash # get rid of old title if exists
```

Comment:
in am gov 2e, not all summaries have a title

This is not related to unit toc so if you prefer I can move out to a different PR 

Also wondering if this is the clearest way to write this?

Reply:
```ruby
summary.first("[data-type='title']")&.trash
```

The `&` is the [safe navigation operator](https://mitrev.net/ruby/2015/11/13/the-operator-in-ruby/) in Ruby.  It'll only call the next method if the part to the left of it is not `nil`.

Fine to have a random change here I think.